manifests/init.pp

```diff
@@ -375,6 +375,9 @@
 #
 # $server_puppetserver_logdir::             The path of the puppetserver log dir
 #
+# $server_puppetserver_logformat::          Format for puppetserver log files, either plain text or json.  Defaults
```

Comment:
This parameter also needs to be added in this file. You can also use the data type `Enum['plain', 'text']` and leave it out of the docs. We also don't include the defaults.